cli/internal/util/cmd.go

```diff
@@ -6,6 +6,18 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// ExitCodeError is a specific error that is returned by the command to specify the exit code
+type ExitCodeError struct {
+	ExitCode int
+}
+
+func (e *ExitCodeError) Error() string { return "exit code error" }
+
+// BasicError is an empty error that is returned by the command
+type BasicError struct{}
```

Comment:
I think we can drop `BasicError` if the above `LogError` doesn't need to return it.

Reply:
How do we return error code 1? Just use ExitCodeError?